app/functions.sh

```diff
@@ -59,8 +78,11 @@ function docker_exec {
     local cmd="${2?missing command}"
     local data=$(printf '{ "AttachStdin": false, "AttachStdout": true, "AttachStderr": true, "Tty":false,"Cmd": %s }' "$cmd")
     exec_id=$(docker_api "/containers/$id/exec" "POST" "$data" | jq -r .Id)
-    if [[ -n "$exec_id" ]]; then
+    if [[ -n $exec_id && ($exec_id != "null") ]]; then
         docker_api /exec/$exec_id/start "POST" '{"Detach": false, "Tty":false}'
+    else
+        echo "$(date), Error: can't exec command ${cmd} in container ${id}. Check container is run" >&2
```

Comment:
`date` output formatting + typo: "Check if the container is running."

Reply:
Fixed